src/metabase/api/card.clj

```diff
@@ -601,16 +601,16 @@ saved later when it is ready."
 (defn- publish-card-update!
   "Publish an event appropriate for the update(s) done to this CARD (`:card-update`, or archiving/unarchiving
   events)."
-  [card archived?]
+  [card-before-updates card-after-update archived?]
```

Comment:
this was a bug, even when we archive/unarchive a card, the event "card-updated" were fired.

with this change it should be correct